contracts/MetadataViews.cdc

```diff
@@ -25,7 +25,7 @@ pub contract MetadataViews {
     /// A group of view resolvers indexed by ID.
     ///
     pub resource interface ResolverCollection {
-        pub fun borrowViewResolver(id: UInt64): &{Resolver}
+        pub fun borrowViewResolver(id: UInt64): &{Resolver}?
```

Comment:
This is a really braking change. But i guess it is needed.

Reply:
Everything here is going to be a breaking change as it is currently designed, so this won't make much of a difference